### ISSUE TITLE
spaceAroundOperator doesn't like shorthand

### ISSUE THREAD

**Issue by tkazec** [lesshint/lesshint#124]
```css
test {
    margin: -5px 0px 5px -5px;
    text-shadow: 0px -1px 0px rgba(0, 0, 0, 0.25);
}
```

```
Warning: test.less: line 2, col 27, spaceAroundOperator: Operators should be preceded and followed by one space.
Warning: test.less: line 3, col 23, spaceAroundOperator: Operators should be preceded and followed by one space.
```

**Comment by jwilsson** [comment]
Looks like our fix in #121 wasn't enough :/ any takers? @miloszsobczak @tkazec?

**Comment by miloszsobczak** [comment]
Shit. This time I will try to make it more flexible and I will look patiently at the whole file.

**Comment by jwilsson** [comment]
Closed by #144.
